Posts/2021/03/File-System-Watcher-Engine-Event.md

```diff
@@ -1,42 +1,54 @@
+---
+post_title: A Reusable File System Event Watcher for PowerShell
+username: msn
+categories: PowerShell
+tags: PowerShell, FileSystemWatcher, Module
+featured_image: 
+summary: FSWatcherEngineEvent module provides events of file system changes as powershell engine event
+---
+
 # A Reusable File System Event Watcher for PowerShell
 
-Have you ever needed a FilesystemWatcher and found it a bit cumbersome to interact with the
-[.Net frameworks FileSystemWatcher class](/dotnet/api/system.io.filesystemwatcher)
-within a powershell script?
+Some time ago I wanted to sync files from a source directory to a destination directory immediately after that had changed in the source directory.
```

Comment:
Some time ago I wanted to sync files from a source directory to a destination directory immediately after they had changed in the source directory.

Reply:
fixed